Geometry/HcalCommonData/src/HcalNumberingFromDDD.cc

```diff
@@ -87,10 +88,10 @@ HcalNumberingFromDDD::HcalID HcalNumberingFromDDD::unitID(int det,
   int    ieta  = hcalConstants->getEta(det, lay, hetaR);
   std::pair<double,double> ficons = hcalConstants->getPhiCons(det, ieta);
 
-  int    nphi  = int((CLHEP::twopi+0.1*ficons.second)/ficons.second);
+  int    nphi  = int((2._pi+0.1*ficons.second)/ficons.second);
```

Comment:
@bsunanda for me this "2._pi" looks terrible. If we introduce DD constants we should introduce namespaces like @VinInn pointed out in other PR and normal constants like DD::twopi

Reply:
This follows several PRs from @cvuosalo (#24509, #25857) using the new user-defined literals feature from C++11: https://en.cppreference.com/w/cpp/language/user_literal

Reply:
@cvuosalo , any constants assumed to be prepared at compilation, so here performance is not an issue. 

I just had a look into the list of literals and it seems to be incomplete: as minimum, ns, GeV and TeV seems to be useful.

Reply:
@civanch: Thanks for the suggestions. I added them to the DDUnits.h issue: #25875.